test/create_db.sh

```diff
@@ -42,6 +39,8 @@ for dbenv in $DBENVS; do
   if [[ ${MYSQL_CONTAINER} ]]; then
     sed -e "s/'localhost'/'%'/g" < ${USERS_SQL} | \
       mysql $dbconn -D $db || die "unable to add users to ${db}"
+  elif mysqld -V | grep "10.0"; then
+      mysql $dbconn -D $db < test/mariadb100_users.sql
```

Comment:
@jsha is this needed now that https://github.com/certbot/certbot/pull/3058 is merged? I think `mariadb100_users.sql` was dropped from the PR and this will break.